### ISSUE TITLE
Three weird bugs on master

### ISSUE THREAD

**Issue by fmaussion** [OGGM/world-glacier-explorer#29]
- the font of the messages above the bars changes after the first language change
- the <br> in the bar about sea-level is ignored
- the colorbar legend does not get updated at translation

![image](https://user-images.githubusercontent.com/10050469/169098335-76d62f87-5627-458e-b4aa-c0a33fe96f7e.png)

@pat-schmitt if you ever are really bored one day...

**Comment by pat-schmitt** [comment]
Yes, will have a look at it later. What do you mean by the second point?

But as a comment for long time deployment we should think about pinning the plotting packages (bokeh, holoviews, panel,...) for the binder environment. As the first and second bugs are probably introduced by some updates in the plotting packages (when starting the app v1.2 on the server from the website (using a docker image in the background) they are not visible). Pinning the packages also means that if we want to use new features of the plotting packages in the future, we need to check for such bugs with the updated versions.

**Comment by fmaussion** [comment]
Yes I know - if you compare the "look" of the two apps (deployed and on binder), there are quite some differences which are hard to track down to a specific action.

I think that what we might need is a deployment on the bokeh server of the current development app, with pinned panel versions indeed (currently the only way to test is locally with a docker container).

I have classes this morning but we can chat at lunch time

**Comment by pat-schmitt** [comment]
I created a small dummy app to extract the python packages out of the docker image for pinning them in environment.yml.
Here is the repository: https://github.com/pat-schmitt/get_docker_package_version
And to run the app with the docker image (here for example `oggm/bokeh:20191210`) you can use this comment in your terminal: `docker run -e BOKEH_ALLOW_WS_ORIGIN=0.0.0.0:8080 -p 8080:8080 oggm/bokeh:20191210 git+https://github.com/pat-schmitt/get_docker_package_version.git app.ipynb` and the app can be found on `http://0.0.0.0:8080/app` in your browser.
Is there a better place to document this?

**Comment by fmaussion** [comment]
Yes - add this to the README?
